.github/workflows/local-wisconsin.yml

```diff
@@ -0,0 +1,137 @@
+name: Wisconsin Local to ECR
+
+on:
+  workflow_dispatch:
+    inputs:
+      site_env:
+        description: "local, pro or user"
+        required: true
+        default: "local"
+      wordpress_url:
+        description: "WP URL"
+        required: true
+        default: "https://dpla.wpengine.com"
+      user_base_url:
+        description: "Local URL"
+        required: true
+        default: "https://recollectionwisconsin.dp.la"
+      pro_base_url:
+        description: "Link to pro site"
+        required: true
+        default: "https://pro.dp.la"
+      pss_base_url:
+        description: "URL for Primary Source Sets"
+        required: true
+        default: "https://dp.la/pssapi"
+      local_id: 
+        description: "Key for local"
+        required: true
+        default: "wisconsin"
+      ga_tracking_id: 
+        description: "Analytics tracking ID"
+        required: true
+        default: "UA-128040088-1"
```

Comment:
I'm a little leery of this being parameterized b/c if we have two locals using the same tracking ID, the stats will be all messed up. Should these all be hardcoded since they're build-specific and each build will be separate?

Reply:
yeah, I agree.